backend/lib/services/controllerregistrations.js

```diff
@@ -30,3 +30,14 @@ exports.list = async function ({ user }) {
   const controllerregistrations = getControllerRegistrations()
   return fromResource(controllerregistrations)
 }
+
+exports.listNetworkingTpes = async function () {
+  const controllerregistrations = getControllerRegistrations()
```

Comment:
Yes, every authenticated used should be able to. I mean - we could also check for the right to create shoots, because otherwise the user doesn't need this information after all. However, idk if we should implement such checks. We discussed this - we agreed, that this is in any case not an ideal solution, as the dashboard backend returns information that a regular (non admin) user is not able to get by accessing the gardener api. However, there is currently no other way...